tests/app/helpers/test_template_helpers.py

```diff
@@ -773,8 +773,18 @@ def test_correct_survey_title_in_context(
             ),
             [{"form_type": "test", "survey_id": "001", "title": "test_title"}],
         ),
-        (SurveyType.HEALTH, "en", QuestionnaireSchema({"survey_id": "001"}), []),
-        (SurveyType.SOCIAL, "en", QuestionnaireSchema({"survey_id": "001"}), []),
+        (
```

Comment:
"After this change, all surveys (due to inheritance) should set survey_id, title and form_type in the data layer if it exists in the schema." but we don't check that for SOCIAL and HEALTH in these tests.